### PR TITLE
EPMRPP-44228 Add Sauce Labs Data centers from plugin details

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/components/integrations/integrationProviders/sauceLabsIntegration/sauceLabsFormFields/sauceLabsFormFields.jsx [hunk 1]
```diff
@@ -56,29 +55,37 @@ export class SauceLabsFormFields extends Component {
...
   };
 
   constructor(props) {
     super(props);
     const {
-      intl: { formatMessage },
+      pluginDetails: { dataCenters = [] },
     } = props;
-    this.dataCenterOptions = [
-      { value: DATA_CENTER_US, label: formatMessage(messages.dataCenterUS) },
```
Comment (ElenaRomanchuk) [inline app/src/components/integrations/integrationProviders/sauceLabsIntegration/sauceLabsFormFields/sauceLabsFormFields.jsx]: Did you forget to remove corresponding messages?

File: app/src/components/integrations/integrationProviders/sauceLabsIntegration/utils.js [hunk 2]
```diff
@@ -14,15 +14,18 @@
...
-  DEFAULT_DATA_CENTER,
-} from './constants';
+import { SAUCE_LABS_DATA_CENTER_ATTRIBUTE_KEY, SAUCE_LABS_ID_ATTRIBUTE_KEY } from './constants';
+
+const getDefaultDataCenter = ({ dataCenters = [''] } = {}) => dataCenters[0];
 
 const getIntegrationDataCenter = (
-  integrations = [{ integrationParameters: { dataCenter: DEFAULT_DATA_CENTER } }],
-) => (integrations[0] && integrations[0].integrationParameters.dataCenter) || DEFAULT_DATA_CENTER;
+  integrations = [{ integrationParameters: { dataCenter: '' }, integrationType: { details: {} } }],
```
Comment (ElenaRomanchuk) [inline app/src/components/integrations/integrationProviders/sauceLabsIntegration/utils.js]: This long default value is redundant. The result of the function would be the same for `integration=[]` default value.

File: app/src/components/integrations/integrationProviders/sauceLabsIntegration/sauceLabsFormFields/sauceLabsFormFields.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/components/integrations/integrationProviders/sauceLabsIntegration/sauceLabsFormFields/sauceLabsFormFields.jsx]: Removed

Comment (AmsterGet) [reply app/src/components/integrations/integrationProviders/sauceLabsIntegration/sauceLabsFormFields/sauceLabsFormFields.jsx]: Fixed

File: app/src/components/integrations/integrationProviders/sauceLabsIntegration/utils.js (hunk 2 above)
Comment (AmsterGet) [reply app/src/components/integrations/integrationProviders/sauceLabsIntegration/utils.js]: Fixed

**Pull request merged**